runtime/onert/core/src/exec/MinMaxData.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_EXEC_MINMAX_DATA_H__
+#define __ONERT_EXEC_MINMAX_DATA_H__
+
+#include "exec/MinMaxMap.h"
+
+#include <string>
+
+namespace onert
+{
+namespace exec
+{
+
+// Because IOMinMaxMap and OpMinMaxMap does not have the ordering and size information,
+// we need to dump model, subgraph id for each minmax
+
+// File structure
+// uint32_t num of runs
```

Comment:
(optional) It may be good to have version of `raw format`.
```suggestion
// uint16_t some magic number
// uint16_t raw schema version
// uint32_t num of runs
```

Reply:
I did not add magic number because it does not have verification function yet. But I also agree to have magic number field. I'll update this later with file reader.